### PR TITLE
add a section on the new melody features from klaus to release notes

### PR DESCRIPTION
**Pull request by jwunderl**
Add a section with some notes on the improvements to the sound & thanks.

(@klausw in case I misrepresented any of the changes, the changes were a little while ago so I might have missed some subtleties when reviewing them. And thanks again!)

### CODE REVIEW

**Review by ganicke — APPROVED** [review]
Thanks for the RTrim on those lines too!

File: docs/blog/arcade/update-jan-2021.md [hunk 1]
```diff
@@ -19,27 +19,39 @@ We all spend a great deal of time in the Image Editor creating the perfect pictu
...
 
 ![The Animation blocks category](/static/blog/arcade/update-jan-2021/animation-category.png)
 
 https://youtu.be/TRjGj80g3Y0
 
-## New Array Block 
+## Sound Effect Enhancements
 
-This has been one of the most requested blocks. In general, randomization is an important aspect of game development, and often times when writing an Arcade game, people want to select a random value from an array. You can of course do this with the "Pick Random" number block for the array indices, but this block makes it so much easier! 
+Sound effects and music are some of the most memorable pieces of many retro videogames, and our support has been improved thanks to contributions from [Klaus W](https://github.com/klausw) over 3 fantastic PRs
```
Comment (ganicke) [inline docs/blog/arcade/update-jan-2021.md:32]:
```suggestion
Sound effects and music are some of the most memorable features of many retro videogames, and our support has been improved thanks to contributions from [Klaus W](https://github.com/klausw) over 3 fantastic PRs:
```

**Comment by klausw** [comment]
Thanks! FYI, I just posted https://github.com/microsoft/pxt-arcade/pull/3002 which adds the new waveforms to the documentation. If that gets merged, maybe consider linking to it from the changelog?

**Review by Jaqster — APPROVED** [review]
Looks good!

**Comment by Jaqster** [comment]
By the way @klausw - my kids and I just spent the last 15 minutes playing around with your new sounds! https://arcade.makecode.com/91960-59425-02296-62940  They're so awesome!!!  THANK YOU :-)

**Comment by jwunderl** [comment]
@klausw That PR looks good too! We working through some build issues right now so might have to wait until tomorrow to get both of these docs changes in, but I've updated the attribution ~

**Comment by klausw** [comment]
> @klausw That PR looks good too! We working through some build issues right now so might have to wait until tomorrow to get both of these docs changes in, but I've updated the attribution ~

Thank you! Just to avoid misunderstandings, I wasn't asking for that change to get called out specifically, what I meant was that it may be useful to add a link to https://arcade.makecode.com/developer/sound as a pointer for people who want to experiment with the additions.

> By the way @klausw - my kids and I just spent the last 15 minutes playing around with your new sounds! https://arcade.makecode.com/91960-59425-02296-62940 They're so awesome!!! THANK YOU :-)

Glad to hear it! I'm currently adding a few more that could be useful for games, I'll update the forum thread once I have something to show.

**Pull request merged**